File: src/entities/Personne.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

//import java.util.Date;

//import java.sql.Date;
import java.util.Date;

/**
 *
 * @author cyrine belhssan
 */
public class Personne {
     private int id;
    private String nom;
    private String prenom;
    private String dateNaissance;
    private String mail;
    private String password;
    private String adresse;
    private int referenceClient;
    private int numero;
    private String civilite;
    private String privilege;
    private String maladie;
    

    public Personne() {
    }

    public Personne(String nom, String prenom, String dateNaissance, String mail, String password, String adresse, int numero, String civilite) {
        this.nom = nom;
        this.prenom = prenom;
        this.dateNaissance = dateNaissance;
        this.mail = mail;
        this.password = password;
        this.adresse = adresse;
        this.numero = numero;
        this.civilite = civilite;
    }
    
    

    public Personne(String nom, String prenom, String dateNaissance, String mail, String password, String privilege) {
        this.nom = nom;
        this.prenom = prenom;
        this.dateNaissance = dateNaissance;
        this.mail = mail;
        this.password = password;
        this.privilege = privilege;
    }

    public Personne(String nom, String prenom, String dateNaissance, String mail, String password, String adresse, int numero, String civilite, String privilege) {
        this.nom = nom;
        this.prenom = prenom;
        this.dateNaissance = dateNaissance;
        this.mail = mail;
        this.password = password;
        this.adresse = adresse;
        this.numero = numero;
        this.civilite = civilite;
        this.privilege = privilege;
    }
    
    

    public Personne(String nom, String prenom, String dateNaissance, String mail, String password, String adresse, int referenceClient, int numero, String civilite, String maladie) {
        this.nom = nom;
        this.prenom = prenom;
        this.dateNaissance = dateNaissance;
        this.mail = mail;
        this.password = password;
        this.adresse = adresse;
        this.referenceClient = referenceClient;
        this.numero = numero;
        this.civilite = civilite;
        this.maladie = maladie;
    }

    public Personne(String nom, String prenom, String mail, String password) {
        this.nom = nom;
        this.prenom = prenom;
        this.mail = mail;
        this.password = password;
    }
    

    public Personne(int id, String nom, String prenom, String dateNaissance, String mail, String password, String adresse, int referenceClient, int numero, String civilite, String privilege, String maladie) {
        this.id = id;
        this.nom = nom;
        this.prenom = prenom;
        this.dateNaissance = dateNaissance;
        this.mail = mail;
        this.password = password;
        this.adresse = adresse;
        this.referenceClient = referenceClient;
        this.numero = numero;
        this.civilite = civilite;
        this.privilege = privilege;
        this.maladie = maladie;
    }
    

    public Personne(int id, String nom, String prenom, String mail, String password, String adresse, int numero, String privilege) {
        this.id = id;
        this.nom = nom;
        this.prenom = prenom;
        this.mail = mail;
        this.password = password;
        this.adresse = adresse;
        this.numero = numero;
        this.privilege = privilege;
    }
    

    public Personne(String nom, String prenom, String dateNaissance, String mail, String password, String adresse, String civilite, String privilege, String maladie) {
        this.nom = nom;
        this.prenom = prenom;
        this.dateNaissance = dateNaissance;
        this.mail = mail;
        this.password = password;
        this.adresse = adresse;
        this.civilite = civilite;
        this.privilege = privilege;
        this.maladie = maladie;
    }
    

    public Personne(String nom, String prenom, String mail, String password, String privilege) {
        this.nom = nom;
        this.prenom = prenom;
        this.mail = mail;
        this.password = password;
        this.privilege = privilege;
    }

    public Personne(String nom, String prenom, String mail, String password, String adresse, int numero, String privilege) {
        this.nom = nom;
        this.prenom = prenom;
        this.mail = mail;
        this.password = password;
        this.adresse = adresse;
        this.numero = numero;
        this.privilege = privilege;
    }

    public Personne(String nom, String prenom, String dateNaissance, String mail, String password, String adresse, int referenceClient, int numero, String civilite, String privilege, String maladie) {
        this.nom = nom;
        this.prenom = prenom;
        this.dateNaissance = dateNaissance;
        this.mail = mail;
        this.password = password;
        this.adresse = adresse;
        this.referenceClient = referenceClient;
        this.numero = numero;
        this.civilite = civilite;
        this.privilege = privilege;
        this.maladie = maladie;
    }

   

    public String getDateNaissance() {
        return dateNaissance;
    }

    public void setDateNaissance(String dateNaissance) {
        this.dateNaissance = dateNaissance;
    }

    

    

    public int getId() {
        return id;
    }

    public String getNom() {
        return nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public String getMail() {
        return mail;
    }

    public String getPassword() {
        return password;
    }

    public String getAdresse() {
        return adresse;
    }

    public int getReferenceClient() {
        return referenceClient;
    }

    public int getNumero() {
        return numero;
    }

    public String getCivilite() {
        return civilite;
    }

    public String getPrivilege() {
        return privilege;
    }

    public String getMaladie() {
        return maladie;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public void setMail(String mail) {
        this.mail = mail;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public void setReferenceClient(int referenceClient) {
        this.referenceClient = referenceClient;
    }

    public void setNumero(int numero) {
        this.numero = numero;
    }

    public void setCivilite(String civilite) {
        this.civilite = civilite;
    }

    public void setPrivilege(String privilege) {
        this.privilege = privilege;
    }

    public void setMaladie(String maladie) {
        this.maladie = maladie;
    }

    @Override
    public String toString() {
        return "Client{" + "id=" + id + ", nom=" + nom + ", prenom=" + prenom + ", mail=" + mail + ", password=" + password + ", adresse=" + adresse + ", referenceClient=" + referenceClient + ", numero=" + numero + ", civilite=" + civilite + ", privilege=" + privilege + ", maladie=" + maladie + '}';
    }
    

    public String AfficherAdmin() {
        return "Admin{" + "id=" + id + ", nom=" + nom + ", prenom=" + prenom + ", mail=" + mail + ", password=" + password + ", privilege=" + privilege +'}';
    } 
    public String AfficherNut() {
        return "Nut{" + "id=" + id + ", nom=" + nom + ", prenom=" + prenom + ", mail=" + mail + ", password=" + password + ", adresse=" + adresse +  ", numero=" + numero + ", civilite=" + civilite + ", privilege=" + privilege + '}';
    }
}

File: src/Entity/Personne.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Personne
 *
 * @ORM\Table(name="personne")
 * @ORM\Entity
 */
class Personne
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="string", length=255, nullable=false)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="prenom", type="string", length=255, nullable=false)
     */
    private $prenom;

    /**
     * @var string|null
     *
     * @ORM\Column(name="dateNaissance", type="string", length=255, nullable=true)
     */
    private $datenaissance;

    /**
     * @var string
     *
     * @ORM\Column(name="mail", type="string", length=255, nullable=false)
     */
    private $mail;

    /**
     * @var string
     *
     * @ORM\Column(name="password", type="string", length=255, nullable=false)
     */
    private $password;

    /**
     * @var string|null
     *
     * @ORM\Column(name="adresse", type="string", length=255, nullable=true)
     */
    private $adresse;

    /**
     * @var int|null
     *
     * @ORM\Column(name="referenceClient", type="integer", nullable=true)
     */
    private $referenceclient;

    /**
     * @var int|null
     *
     * @ORM\Column(name="numero", type="integer", nullable=true)
     */
    private $numero;

    /**
     * @var string|null
     *
     * @ORM\Column(name="civilite", type="string", length=255, nullable=true)
     */
    private $civilite;

    /**
     * @var string
     *
     * @ORM\Column(name="privilege", type="string", length=255, nullable=false)
     */
    private $privilege;

    /**
     * @var string|null
     *
     * @ORM\Column(name="maladie", type="string", length=255, nullable=true)
     */
    private $maladie;

    /**
     * @var int|null
     *
     * @ORM\Column(name="mode", type="integer", nullable=true)
     */
    private $mode;

    /**
     * @var string|null
     *
     * @ORM\Column(name="image", type="string", length=255, nullable=true)
     */
    private $image;

    /**
     * @return int
     */
    public function getId(): int
    {
        return $this->id;
    }

    /**
     * @param int $id
     * @return Personne
     */
    public function setId(int $id): Personne
    {
        $this->id = $id;
        return $this;
    }

    /**
     * @return string
     */
    public function getNom(): ?string
    {
        return $this->nom;
    }

    /**
     * @param string $nom
     * @return Personne
     */
    public function setNom(string $nom): Personne
    {
        $this->nom = $nom;
        return $this;
    }

    /**
     * @return string
     */
    public function getPrenom(): ?string
    {
        return $this->prenom;
    }

    /**
     * @param string $prenom
     * @return Personne
     */
    public function setPrenom(string $prenom): Personne
    {
        $this->prenom = $prenom;
        return $this;
    }

    /**
     * @return string|null
     */
    public function getDatenaissance(): ?string
    {
        return $this->datenaissance;
    }

    /**
     * @param string|null $datenaissance
     * @return Personne
     */
    public function setDatenaissance(?string $datenaissance): Personne
    {
        $this->datenaissance = $datenaissance;
        return $this;
    }

    /**
     * @return string
     */
    public function getMail(): ?string
    {
        return $this->mail;
    }

    /**
     * @param string $mail
     * @return Personne
     */
    public function setMail(string $mail): Personne
    {
        $this->mail = $mail;
        return $this;
    }

    /**
     * @return string
     */
    public function getPassword(): ?string
    {
        return $this->password;
    }

    /**
     * @param string $password
     * @return Personne
     */
    public function setPassword(?string $password): Personne
    {
        $this->password = $password;
        return $this;
    }

    /**
     * @return string|null
     */
    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    /**
     * @param string|null $adresse
     * @return Personne
     */
    public function setAdresse(?string $adresse): Personne
    {
        $this->adresse = $adresse;
        return $this;
    }

    /**
     * @return int|null
     */
    public function getReferenceclient(): ?int
    {
        return $this->referenceclient;
    }

    /**
     * @param int|null $referenceclient
     * @return Personne
     */
    public function setReferenceclient(?int $referenceclient): Personne
    {
        $this->referenceclient = $referenceclient;
        return $this;
    }

    /**
     * @return int|null
     */
    public function getNumero(): ?int
    {
        return $this->numero;
    }

    /**
     * @param int|null $numero
     * @return Personne
     */
    public function setNumero(?int $numero): Personne
    {
        $this->numero = $numero;
        return $this;
    }

    /**
     * @return string|null
     */
    public function getCivilite(): ?string
    {
        return $this->civilite;
    }

    /**
     * @param string|null $civilite
     * @return Personne
     */
    public function setCivilite(?string $civilite): Personne
    {
        $this->civilite = $civilite;
        return $this;
    }

    /**
     * @return string
     */
    public function getPrivilege(): ?string
    {
        return $this->privilege;
    }

    /**
     * @param string $privilege
     * @return Personne
     */
    public function setPrivilege(string $privilege): Personne
    {
        $this->privilege = $privilege;
        return $this;
    }

    /**
     * @return string|null
     */
    public function getMaladie(): ?string
    {
        return $this->maladie;
    }

    /**
     * @param string|null $maladie
     * @return Personne
     */
    public function setMaladie(?string $maladie): Personne
    {
        $this->maladie = $maladie;
        return $this;
    }

    /**
     * @return int|null
     */
    public function getMode(): ?int
    {
        return $this->mode;
    }

    /**
     * @param int|null $mode
     * @return Personne
     */
    public function setMode(?int $mode): Personne
    {
        $this->mode = $mode;
        return $this;
    }

    /**
     * @return string|null
     */
    public function getImage(): ?string
    {
        return $this->image;
    }

    /**
     * @param string|null $image
     * @return Personne
     */
    public function setImage(?string $image): Personne
    {
        $this->image = $image;
        return $this;
    }


}
